unitTests/ui/DynamicFacet/DynamicFacetTestUtils.ts

```diff
@@ -6,12 +6,16 @@ import * as Mock from '../../MockEnvironment';
 import { IFacetResponse } from '../../../src/rest/Facet/FacetResponse';
 
 export class DynamicFacetTestUtils {
+  static get defaultOptions() {
+    return {
+      field: '@dummy'
+    };
+  }
+
   static createFakeFacet(options?: IDynamicFacetOptions) {
     const facet = Mock.mockComponent<DynamicFacet>(DynamicFacet);
     facet.options = {
-      id: 'dummy',
-      field: '@dummy',
-      title: 'a title',
+      ...this.defaultOptions,
```

Comment:
Since the defaultOptions/Options destructuring is happening here and on line 32, it would be better to create a function `allOptions` and call it on both lines.